networking_generic_switch/devices/corsa_devices/__init__.py

```diff
@@ -115,10 +115,18 @@ def add_network(self, segmentation_id, network_id, of_controller=None):
             cont_ip, cont_port = of_controller
             if 'controllerNamespace' in self.config:
                 c_controller_namespace = self.config['controllerNamespace']
-  
+
+        isVFCHost = True
+        if not 'VFCHost' in self.config or not self.config['VFCHost'] == 'True':
+            LOG.info("PRUTH: Skipping VFC Creation: isVFCHost " + str(isVFCHost))
+            isVFCHost = False
+            return
```

Comment:
Given the return here, it looks like there is no need to have the `isVFCHost` variable set, as it will always be True if it makes it past this line, therefore it is tautological.

Reply:
What you say is correct.  At least for now, there is nothing to do if it is not a VFC host. We were thinking that we might dynamically plumb the VLANs on those switches but ultimatly decided to statically provision them.